chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -322,5 +311,14 @@ def make_zone(zone)
   notifies :restart, "service[bind9]", :immediately
```

Comment:
It should work fine (because you used the string way to reference the service), but wanted to double-check :-)